provisioning/service/src/Manager/IndividualEnrollmentManager.cs

```diff
@@ -71,13 +73,21 @@ internal static class IndividualEnrollmentManager
             IEnumerable<IndividualEnrollment> individualEnrollments,
             CancellationToken cancellationToken)
         {
-            //TODO: Implement.
-
-            /* SRS_INDIVIDUAL_ENROLLMENT_MANAGER_21_004: [The BulkOperationAsync shall throw ArgumentException if the provided individualEnrollments is null.] */
-            /* SRS_INDIVIDUAL_ENROLLMENT_MANAGER_21_005: [The BulkOperationAsync shall sent the Put Http request to run the bulk operation to the collection of the individualEnrollment.] */
-            /* SRS_INDIVIDUAL_ENROLLMENT_MANAGER_21_006: [The BulkOperationAsync shall return an BulkEnrollmentOperationResult object created from the body of the Http response.] */
+            /* SRS_INDIVIDUAL_ENROLLMENT_MANAGER_21_004: [The BulkOperationAsync shall throw ArgumentException if the provided 
+                                                    individualEnrollments is null or empty.] */
+            if (!(individualEnrollments ?? throw new ArgumentNullException("individualEnrollments")).Any())
```

Comment:
Nit: same as above - this format makes it less clear that the purpose is the .Any() call.

Reply:
Keep ??, at least for now